src/main/java/TextController.java

```diff
@@ -87,9 +87,7 @@ public void run() {
         Scanner reader = new Scanner(System.in);
 
         while(true) {
-            System.out.println("Awaiting user input");
```

Comment:
- [ ] I think we should have some sort of a prompt for user input. Something like `System.out.print(">> ");`